worlds/sand_island.world

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" ?>
+<sdf version="1.4">
+  <world name="default">
+    <scene>
+      <sky>
+        <clouds>
+          <speed>12</speed>
+        </clouds>
+      </sky>
+    </scene>
+    <!-- A global light source -->
```

Comment:
As tfoote mentioned, can you add the location with the `spherical_coordinate` tag?

Reply:
I'm not able to find the original coordinates

Reply:
> I'm not able to find the original coordinates

@tfoote can you help here?

Reply:
We don't have high precision for this one. The other ones were mostly hand tuned by registering qgc and gazebo visualizations and flying the drones to observable landmarks.

This is a less interesting world and we didn't do the full integration into our demo with it. Which is when I did the high precision registration. Unless there's strong interest in some particular aspect I'd suggest we just close this PR and focus on the other more featureful worlds.

Reply:
What about just making sure that the drone is spawned at the terrain level?

We can focus on cleaning the others but I think this one can be kept as well.

Reply:
@ahcorde @tfoote Are there any updates?